zgw/documenten-api/src/main/kotlin/com/ritense/portal/documentenapi/client/ClamAVVirusScanConfig.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ritense.portal.documentenapi.client
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+@ConfigurationProperties(prefix = "valtimo.zgw.documentenapi.virusscan.clamav")
+data class ClamAVVirusScanConfig(
+    var hostName: String = "",
```

Comment:
Graag `val` gebruiken, en de lege string achterwege laten. Als de hostname optioneel is, graag nullable maken.